app/models/manageiq/providers/amazon/instance_types.rb

```diff
@@ -13,6 +13,30 @@
 module ManageIQ::Providers::Amazon::InstanceTypes
   # Types that are currently advertised for use
   AVAILABLE_TYPES = {
+    "t2.nano"     => {
+      :name                    => "t2.nano",
+      :family                  => "General Purpose",
+      :description             => "T2 Nano",
+      :memory                  => 0.5.gigabyte,
+      :vcpu                    => 1,
+      :ebs_only                => true,
+      :instance_store_size     => 0,
+      :instance_store_volumes  => 0,
+      :architecture            => [:i386, :x86_64],
+      :virtualization_type     => [:hvm],
+      :network_performance     => :low,
```

Comment:
Yeah. I noticed that but leaned on the side of caution and went with low. I've actually found it documented as `low_to_moderate`  in one place and `low` in another. I thought the safest thing was to go with low.